src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -694,13 +697,28 @@ export class PageControllerBase {
     if (feedbackContextInfo) {
       viewModel.name = feedbackContextInfo.formTitle
     }
+
+    let feedbackLink: string | undefined = ''
+
     // setting the feedbackLink to undefined here for feedback forms prevents the feedback link from being shown
     if (this.def.feedback?.url) {
-      viewModel.feedbackLink = this.feedbackUrlFromRequest(request)
+      feedbackLink = this.feedbackUrlFromRequest(request)
+    } else if (this.def.feedback?.emailAddress) {
+      feedbackLink = `mailto:${this.def.feedback.emailAddress}`
+    } else {
+      feedbackLink = config.feedbackLink
     }
-    if (this.def.feedback?.emailAddress) {
-      viewModel.feedbackLink = `mailto:${this.def.feedback.emailAddress}`
+
+    if (feedbackLink?.startsWith('mailto:')) {
```

Comment:
Mind if we add tests into **PageControllerBase.test.ts** for this?

Would be ace to clearly see the escaping + error logging in action as currently missed these checks from the summary page view model ([see comment](https://github.com/DEFRA/forms-runner/pull/220/files#r1662041535))

Probably seen this, but we'll want a Jest env adding for `feedbackLink` so the unit tests run too

```console
The server config is invalid. "feedbackLink" is required
```

Reply:
Ah I missed this, thanks, I'll add it.

Reply:
I've done some testing and we should escape `http(s)://` URLs too

```js
    if (feedbackLink) {
      const isExternal = /^(https?|mailto):/.test(feedbackLink)
      const baseURL = !isExternal ? config.get('baseURL') : undefined // Need APP_BASE_URL in config

      try {
        feedbackLink = new URL(feedbackLink, baseURL).toString()
      } catch (err) {
        logger.error(err, `Failed to decode ${feedbackLink}`)
        feedbackLink = undefined
      }
    }
```